hanami-cli.gemspec

```diff
@@ -30,4 +30,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "bundler", ">= 1.6", "< 3"
   spec.add_development_dependency "rake",  "~> 12.0"
   spec.add_development_dependency "rspec", "~> 3.7"
+  spec.add_development_dependency "rubocop"
+  spec.add_development_dependency "rainbow", '=2.2.2'
```

Comment:
```suggestion
```
Unless there's some reason we need this gem / version.

Reply:
See below.